packages/snaps-sdk/src/jsx/jsx-dev-runtime.ts

```diff
@@ -0,0 +1,27 @@
+import type { JsonObject, Key, SnapComponent } from './component';
+import { jsx } from './jsx-runtime';
+import { assertJSXElement } from './validation';
+
+/**
+ * The JSX runtime for Snaps SDK components. This function is used to render
+ * Snap components into a format that can be used by the Snaps.
+ *
+ * This is the "development" version of the runtime, which includes additional
+ * validation, which is otherwise handled by MetaMask. To use this runtime,
+ * specify `@metamask/snaps-sdk` as import source for JSX, and use
+ * `react-jsxdev` as the pragma.
+ *
+ * @param component - The component to render.
+ * @param props - The props to pass to the component.
+ * @returns The rendered component.
+ * @see https://www.typescriptlang.org/tsconfig/#jsx
+ */
+export function jsxDEV<Props extends JsonObject>(
```

Comment:
Do we not need a `jsxs`?

Reply:
Apparently not, `react-jsxdev` always uses `jsxDEV`:

https://github.com/facebook/react/blob/190cc990e01e5131a6b26f1a3212444cebd794e2/packages/react/jsx-dev-runtime.js